client/packages/invoices/src/Prescriptions/DetailView/DetailView.tsx

```diff
@@ -77,7 +93,16 @@ export const PrescriptionDetailView: FC = () => {
             },
           })}
         >
-          <AppBarButtons onAddItem={onAddItem} />
+          <AppBarButtons onAddItem={onAddItem} onViewHistory={onViewHistory} />
+          {isHistoryOpen && (
```

Comment:
I feel like I've had issues in the past with this pattern, but can't remember the details. I think it caused a problem in Health Supply Hub where modals weren't opening correctly for some reason.

We don't actually need to have the conditional here right? Does this reduce the render effort or something?

I'm ok to keep it but thought I'd just query it.

Reply:
No, you're right, it's unecessary -- another one of Chris's leftovers that I didn't really consider :) 7870fbbc4c